R/choices_labeled.R

```diff
@@ -1,3 +1,4 @@
+# TODO: this whole file needs work
```

Comment:
if this is still applicable - would you mind to create issue instead of putting such comment?
I would like to have everything to be done as an issue instead of scraping the codebase. This is way better to manage.